src/main/java/it/pagopa/interop/probing/caller/consumer/PollingReceiver.java

```diff
@@ -0,0 +1,37 @@
+package it.pagopa.interop.probing.caller.consumer;
+
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.awspring.cloud.messaging.listener.SqsMessageDeletionPolicy;
+import io.awspring.cloud.messaging.listener.annotation.SqsListener;
+import it.pagopa.interop.probing.caller.dto.EserviceContent;
+import it.pagopa.interop.probing.caller.util.EserviceTechnology;
+import it.pagopa.interop.probing.caller.util.logging.Logger;
+
+
+@Component
+public class PollingReceiver {
+
+  @Autowired
+  ObjectMapper mapper;
+
+  @Autowired
+  private Logger logger;
+
+  @SqsListener(value = "${amazon.sqs.end-point.poll-queue}",
+      deletionPolicy = SqsMessageDeletionPolicy.ON_SUCCESS)
+  public void receiveStringMessage(final String message) throws IOException {
+
+    EserviceContent service = mapper.readValue(message, EserviceContent.class);
+
+    logger.logMessageReceiver(service.getId());
+
+    if (EserviceTechnology.REST.getValue().equals(service.getTechnology().getValue())) {
```

Comment:
???

Reply:
useful for future developments, but i will delete it for now